### PR TITLE
coredumpctl: add page

### PR DESCRIPTION
**Pull request by jensecj**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]

File: pages/linux/coredumpctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`coredumpctl info {{PID}}`
+
+- Invoke debugger using the last core dump of a program:
+
+`coredumpctl debug {{program}}`
+
+- Extract the last core dump of a program to a file:
+
+`coredumpctl -o {{path/to/file}} dump {{program}}`
```
Comment (bl-ue) [inline pages/linux/coredumpctl.md:24]: We prefer long options when possible, @jensecj.
```suggestion
`coredumpctl --output={{path/to/file}} dump {{program}}`
```

**Comment by bl-ue** [comment]
@jensecj please try to avoid force-pushing when possible, as we automatically squash commits before merging and it can be reviewing a bit difficult 🙂

**Review by bl-ue — APPROVED** [review]
Thank you @jensecj!

Welcome to tldr-pages 😀

**Comment by jensecj** [comment]
Alright :smile: and cheers for the incredibly fast turnaround!

**Pull request merged**